### ISSUE TITLE
Bin ranges have weird format

### ISSUE THREAD

**Issue by domoritz** [vega/vega-lite#1757]
```json
{
  "description": "A scatterplot showing horsepower and miles per gallons with binned acceleration on opacity.",
  "data": {"url": "data/cars.json"},
  "mark": "point",
  "encoding": {
    "x": {"field": "Horsepower","type": "quantitative"},
    "y": {"field": "Miles_per_Gallon","type": "quantitative"},
    "color": {"bin": true, "field": "Acceleration","type": "quantitative"}
  }
}
```

<img width="355" alt="screen shot 2016-12-03 at 14 39 54" src="https://cloud.githubusercontent.com/assets/589034/20862678/60478110-b966-11e6-8011-02aada599b42.png">

I think we can just remove the formatting since binning schould produce reasonable bounds anyway (by default).

**Comment by kanitw** [comment]
>  binning schould produce reasonable bounds anyway.

What happens when we bin billions? 

I wonder if we should fix #1539 before dealing with this (or at least consider this and #1539 together).

Totally always remove format from binning seems too opinionated since sometimes users might want to customize?

**Comment by kanitw** [comment]
As of #2254, this seems fixed. 
![vega_editor](https://cloud.githubusercontent.com/assets/111269/25300875/28b89be6-26ce-11e7-9be1-25975a3ce020.png)
